### PR TITLE
feat(hybrid-cloud): Break inviter foreign key on inviters for OrganizationMember model

### PR DESCRIPTION
**Pull request by dashed**

### CODE REVIEW

File: src/sentry/api/serializers/models/organization_member/base.py [hunk 1]
```diff
@@ -72,7 +71,19 @@ def get_attrs(
         users_by_id: Mapping[str, Any] = {
             u["id"]: u for u in user_service.serialize_many(filter={"user_ids": users_set})
         }
-        actor_ids = [u.actor_id for u in user_service.get_many(filter={"user_ids": users_set})]
+        actor_ids = [
+            u.actor_id for u in user_service.get_many(filter={"user_ids": users_set}) if u.actor_id
+        ]
```
Comment (markstory) [inline src/sentry/api/serializers/models/organization_member/base.py:76]: We'll also need to cut out `user.actor_id` usage as that column is being removed. If we only need the actor ids for a set of users we could do 

```python
actors_ids = Actor.objects.filter(type=ACTOR_TYPES["user"], user_ids__in=users_set).values_list("id", flat=True)
```

To get the relevant actor records for the users in the current region.

File: src/sentry/api/serializers/models/organization_member/base.py [hunk 2]
```diff
@@ -72,7 +71,19 @@ def get_attrs(
...
+        inviters_set = sorted(
+            {
+                organization_member.inviter_id
+                for organization_member in item_list
+                if organization_member.inviter_id
+            }
+        )
+        inviters_by_id: Mapping[int, Any] = {
```
Comment (markstory) [inline src/sentry/api/serializers/models/organization_member/base.py:84]:
```suggestion
        inviters_by_id: Mapping[int, RpcUser] = {
```

**Review by wedamija — COMMENTED** [review]
Migration lgtm. Is this the last foreign key to break?

**Review by markstory — APPROVED** [review]
Looks good to me.

**Pull request merged**